config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "/customers/:id", to: "customers#show"
+  post "/customers/:id", to: "customer_items#create"
```

Comment:
To make this even more RESTful we would want to add `items` to the end of the URL
```ruby
post "/customers/:id/items", to: "customer_items#create"
```